app.py

```diff
@@ -93,6 +97,14 @@ def invalid_request(e):
     return jsonify(error=400, text=str(e)), 400
 
 
+@jwt.token_in_blacklist_loader
```

Comment:
also we need to add some variables in config
![image](https://user-images.githubusercontent.com/30180012/76063174-3e1d5980-5f98-11ea-8e7c-c5c4e49b91a4.png)